articlequality/feature_lists/property_datatypes.py

```diff
@@ -0,0 +1,2564 @@
+    'P6524',
+    'P6530',
+    'P6531',
+    'P6532',
+    'P6533',
+    'P6534',
+    'P6540',
+    'P6563',
+    'P6568',
+    'P6569',
+    'P6587',
+    'P6606',
+    'P6608',
+    'P6657',
+    'P6658',
+    'P6684',
+    'P6718',
+    'P6731',
+    'P6758',
+    'P6803',
+    'P6819',
+    'P6840',
+    'P6855',
+    'P6872',
+    'P6875',
+    'P6884',
+    'P6885',
+    'P6886',
+    'P6887',
+    'P6889',
+    'P6902',
+    'P6938',
+    'P6939',
+    'P6942',
+    'P6948',
+    'P6954',
+    'P6962',
+    'P6977',
+    'P6978',
+    'P7010',
+    'P7045',
+    'P7047',
+    'P7075',
+    'P7078',
+    'P7084',
+    'P7086',
+    'P7087',
+    'P7095',
+    'P7108',
+    'P7122',
+    'P7137',
+    'P7152',
+    'P7153',
+    'P7160',
+    'P7162',
+    'P7163',
+    'P7165',
+    'P7167',
+    'P7169',
+    'P7174',
+    'P7209',
+    'P7228',
+    'P7252',
+    'P7253',
+    'P7261',
+    'P7309',
+    'P7327',
+    'P7367',
+    'P7374',
+    'P7376',
+    'P7377',
+    'P7378',
+    'P7406',
+    'P7419',
+    'P7442',
+    'P7452',
+    'P7469',
+    'P7479',
+    'P7481',
+    'P7482',
+    'P7486',
+    'P7500',
+    'P7501',
+    'P7508',
+    'P7514',
+    'P7528',
+    'P7561',
+    'P7573',
+    'P7582',
+    'P7600',
+    'P7601',
+    'P7603',
+    'P7604',
+    'P7643',
+    'P7719',
+    'P7727',
+    'P7763',
+    'P7767',
+    'P7779',
+    'P7781',
+    'P7782',
+    'P7861',
+    'P7867',
+    'P7888',
+    'P7903',
+    'P7904',
+    'P7936',
+    'P7937',
+    'P7938',
+    'P7959',
+    'P7984',
+    'P8001',
+    'P8004',
+    'P8005',
+    'P8006',
+    'P8026',
+    'P8030',
+    'P8031',
+    'P8032',
+    'P8045',
+    'P8047',
+    'P8058',
+    'P8097',
+    'P8107',
+    'P8111',
+    'P8115',
+    'P8127',
+    'P8131',
+    'P8138',
+    'P8193',
+    'P8194',
+    'P8196',
+    'P8203',
+    'P8225',
+    'P8228',
+    'P8241',
+    'P8258',
+    'P8263',
+    'P8274',
+    'P8289',
+    'P8306',
+    'P8307',
+    'P8308',
+    'P8324',
+    'P8327',
+    'P8329',
+    'P8331',
+    'P8339',
+    'P8345',
+    'P8363',
+    'P8371',
+    'P8395',
+    'P8397',
+    'P8402',
+    'P8403',
+    'P8411',
+    'P8413',
+    'P8423',
+    'P8428',
+    'P8431',
+    'P8450',
+    'P8453',
+    'P8459',
+    'P8464',
+    'P8480',
+    'P8500',
+    'P8503',
+    'P8535',
+    'P8536',
+    'P8546',
+    'P5188',
+    'P5191',
+    'P5192',
+    'P5238',
+    'P5401',
+    'P5402',
+    'P5920',
+    'P5978',
+    'P6254',
+    'P6553',
+    'P6571',
+    'P6712',
+    'P7706',
+    'P1647',
+    'P1659',
+    'P1687',
+    'P1696',
+    'P1963',
+    'P2271',
+    'P2304',
+    'P2306',
+    'P2368',
+    'P3176',
+    'P4155',
+    'P6609',
+    'P6824',
+    'P8379',
+    'P8396',
+    'P5972',
+    'P5973',
+    'P5974',
+    'P5975',
+    'P5976',
+    'P5977',
+    'P5979',
+    'P5980',
+    'P6072',
+    'P6593',
+    'P6719',
+    'P7018',
+    'P7219',
+    'P7220',
+    'P7221',
+     'P8471']
```

Comment:
I think at this size, this would work much better as a data file that gets imported.  You could write a script for regenerating it as need.

Reply:
I just realized this file (module) is inside "feature_lists" but it doesn't contain any features.  It seems like this belongs in wikidata.py (possibly imported from a data file)  We follow this pattern in revscoring.  See https://github.com/wikimedia/revscoring/blob/master/revscoring/languages/english.py#L278 as an example that uses python asset management ("resources").